spec/support/json_helper.rb

```diff
@@ -5,15 +5,25 @@ module JsonHelper
   def json
     @json ||= begin
       body = response.body
-      JSON.parse(body) if body.present?
+
+      if response.status == 204
+        return {}
+      elsif body.blank?
```

Comment:
This as well we can ignore since it is a helper for specs to catch if for some reason response body is missing in specs.
It is not a production code.